site/zenodo_rdm/previewer/mirador.py

```diff
@@ -64,6 +55,46 @@ def preview(file):
         if tile_file is not None and tile_file.processor:
             tile_status = tile_file.processor.get("status")
 
+    # Check if the file was updated more than an hour ago
+    last_updated_time = datetime.fromisoformat(file.data["updated"])
+    current_time = datetime.now(timezone.utc)
+    time_diff = current_time - last_updated_time
+
+    # If the image tile status size is processing and the image was last updated more than an hour ago
+    if tile_status == "processing" and time_diff > timedelta(hours=1):
```

Comment:
Should `timedelta(hours=1)` be placed in the config?

Reply:
Naming committee decided the config is:

```python
PREVIEWER_IMAGE_FAILED_PROCESSING_TIMEDELTA = {
    "hours": 1,
}
```